### ISSUE TITLE
Windows fusion dev / yarn dev not working

### ISSUE THREAD

**Issue by JarredMiles** [fusionjs/fusion-cli#461]
### Type of issue

Bug

### Description

When using yarn dev / fusion dev on a windows machine I get an error from the project on the getting started page
### Current behavior
[the issue](https://i.imgur.com/k81iNuK.png)
[stacktrace](https://i.imgur.com/hDuS2gu.png)
### Expected behavior

slashes need to be double \\\ C:\\\path\\\to\\\project
I expect to run yarn dev without error

### Steps to reproduce
On a windows machine do the following 2 commands
1. yarn create fusion-app my-fusion-app
2. yarn dev

### Your environment

* fusion-cli version: 1.8.4

* Node.js version (`node --version`): 8.9.4

* npm version (`npm --version`): 5.6.0

* Operating System: Windows 10 64bit

**Comment by brandoncc** [comment]
I researched this and found the same issue today.  I believe the actual problem is occurring when webpack outputs the server.js bundle file. The issue is [here](https://github.com/fusionjs/fusion-cli/blob/master/build/compiler.js#L479-L487).

`require.resolve` is properly escaping the slashes, but then webpack outputs the escaped slashes to the bundle file which swallows them. I verified that by changing the banner to:

```
banner: `"c:\\dd"`
```

and the server.js file contained:

```
"c:\dd"
```

I have come up with two solutions. One is better than the other, but I'm not sure about either of them. The first is pretty bad, but worked in my tests:

```
banner: `require("${require.resolve(
  'source-map-support'
).replace(/\\/g, "\\\\")}").install();`,
```

This double-escapes backslashes, so that by the end of the pipeline the proper amount exist. The other solution uses [upath](https://github.com/anodynos/upath). Upath is already in the project as a nested dependency and could be surfaced as a primary dependency:

```bash
$ yarn why upath
yarn why v1.7.0
warning package.json: No license field
[1/4] 🤔  Why do we have the module "upath"...?
[2/4] 🚚  Initialising dependency graph...
warning example-fusion-app@0.0.0: No license field
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "upath@1.1.0"
info Reasons this module exists
   - "fusion-cli#webpack#watchpack#chokidar" depends on it
   - Hoisted from "fusion-cli#webpack#watchpack#chokidar#upath"
info Disk size without dependencies: "56KB"
info Disk size with unique dependencies: "56KB"
info Disk size with transitive dependencies: "56KB"
info Number of shared dependencies: 0
✨  Done in 0.65s.
```

Since all node commands can use forward slashes on Windows, upath replaces backslashes with forward slashes. This fixes the issue described here. This is the code to achieve it:

```
banner: `require("${upath.normalize(require.resolve(
  'source-map-support'
))}").install();`,
```

My only problem with this is that it feels like a bandaid but I'm not sure it is a true fix. By that I mean that it is a working solution, but if you look at the changed code, is it obvious what it fixes and why? It is possible that a simple comment is enough to describe the problem and solution, but I'm wondering if this solution is targeted too much at the edge case and there is a better overall solution.

**Comment by KevinGrandon** [comment]
Hi folks. Wondering if anyone developing on windows has docker experience or would be interested in helping to maintain a docker + Fusion.js boilerplate. A single reproducible environment within a docker container might be the best approach here to ensure we have great support for everyone.

**Comment by KevinGrandon** [comment]
Update: I've created a simple Fusion.js boilerplate /w Docker that we should be able to work on Windows. Looking for someone to help me verify that this works, it should just be a single command. Steps:

* Clone or download this folder: https://github.com/KevinGrandon/fusion-boilerplate/tree/master/docker-fusion-app
* Run: `docker-compose up`

**Comment by JarredMiles** [comment]
Sorry about the late response. I did try running it, but it did not work for me stating that it couldn't pull in the package.json and yarn.lock. I haven't had the time to fiddle with it recently, but when I do I'll report back here.

**Comment by jidemobell** [comment]
sorry about the late response as well. I will give it a check

**Comment by maihungdev** [comment]
> Update: I've created a simple Fusion.js boilerplate /w Docker that we should be able to work on Windows. Looking for someone to help me verify that this works, it should just be a single command. Steps:
> 
> * Clone or download this folder: https://github.com/KevinGrandon/fusion-boilerplate/tree/master/docker-fusion-app
> * Run: `docker-compose up`

can't find module. same issue.

**Comment by chrisdothtml** [comment]
This is fixed and has been prereleased as `1.18.0-alpha.1`. It will also be included the next release sometime next week
